src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibratorIntegrationTest.java

```diff
@@ -98,7 +97,36 @@ public void testVariantRecalibratorSNP(final String[] params) throws IOException
         // the expected vcf is not in the expected dir because its used
         // as input for the ApplyVQSR test
         IntegrationTestSpec.assertEqualTextFiles(recalOut, new File(getLargeVQSRTestDataDir() + "snpRecal.vcf"));
-        IntegrationTestSpec.assertEqualTextFiles(tranchesOut, new File(getLargeVQSRTestDataDir() + "expected/SNPTranches.txt"));
+        IntegrationTestSpec.assertEqualTextFiles(tranchesOut, new File(expectedTranchesFile));
+    }
+
+    @Test(dataProvider = "VarRecalSNP")
+    public void testVariantRecalibratorSNP(final String[] params) throws IOException {
+        doSNPTest(params, getLargeVQSRTestDataDir() + "expected/SNPTranches.txt");
+    }
+
+    @Test(dataProvider = "VarRecalSNP")
+    public void testVariantRecalibratorSNPWithTranches(final String[] params) throws IOException {
+        // same as testVariantRecalibratorSNP but with specific tranches
+        List<String> args = new ArrayList<>(params.length);
+        Stream.of(params).forEach(arg -> args.add(arg));
+        args.addAll(
+                Arrays.asList(
+                        "-tranche", "100.0",
+                        "-tranche", "99.95",
+                        "-tranche", "99.9",
+                        "-tranche", "99.8",
+                        "-tranche", "99.6",
+                        "-tranche", "99.5",
+                        "-tranche", "99.4",
+                        "-tranche", "99.3",
+                        "-tranche", "99.0",
+                        "-tranche", "98.0",
+                        "-tranche", "97.0",
+                        "-tranche", "90.0"
+                )
+        );
+        doSNPTest(args.toArray(new String[args.size()]), getLargeVQSRTestDataDir() + "expected/SNPWithTranches.txt");
```

Comment:
The expected output here is named `SNPWithTranches.txt`, which is fine -- but the expected output for the test without explicit tranches is named `SNPTranches.txt`, which is confusing.